src/test/resources/gff3toff_rules/unknown_strand.gff3

```diff
@@ -0,0 +1,3 @@
+##gff-version 3.1.26
+##sequence-region TEST01.1 1 6107
+TEST01.1	.	repeat_region	77	138	.	?	.	note=L2;
```

Comment:
Can you also check the following ENSEMBL region:
`1	ilAbrTril1.1	region	1	15215371	.	.	.	ID=region:1;Alias=OX276421.1`